arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -2997,23 +2997,26 @@ void IResearchAnalyzerFeature::cleanupAnalyzers(irs::string_ref database) {
 void IResearchAnalyzerFeature::invalidate(const TRI_vocbase_t& vocbase) {
   WRITE_LOCKER(lock, _mutex);
   auto database = irs::string_ref(vocbase.name());
-  auto itr = _lastLoad.find(
-      static_cast<std::string>(  // FIXME: after C++20 remove cast and use
-                                 // heterogeneous lookup
-          irs::make_hashed_ref(database, std::hash<irs::string_ref>())));
+  // FIXME: after C++20 remove cast and use
+  // heterogeneous lookup
+  auto itr = _lastLoad.find(static_cast<std::string>(
+      irs::make_hashed_ref(database, std::hash<irs::string_ref>())));
```

Comment:
Still doesn't work?
Maybe we should remove comment in that case?
Or use another map type